lib/README.md

```diff
@@ -127,19 +126,32 @@ curl -H "Authorization: Bearer ${JWT}" http://localhost:8000/flows/v1/node -H 'c
 ```
 Now the node will be available on the FlowBroker Dojot's interface.
 
-#### Making changes to the source code of the node and seeing them reflected
+#### Live Reloading your node
 
-**Note:** If you need to the source code of the node and seeing them reflected, you need to: **rebuild the container**; **push it to the registry again with a new tag**; **remove the node from dojot via api**; **and add the node to dojot again**. Always change the `<image tag>` of the docker image, to force the update and see your changes reflected.
+**Note:** If you need to change the source code of the node and seeing them reflected, you need to: **rebuild the container**; **push it to the registry again with a new tag**; **remove the node from dojot via api**; **and add the node to dojot again**. Always change the `<image tag>` of the docker image, to force the update and see your changes reflected.
 
 How to remove a node from Dojot:
 
 ```sh
 curl -X DELETE -H "Authorization: Bearer ${JWT}" http://localhost:8000//flows/v1/node/<node name> -H 'content-type: application/json'
 ```
 
-**ATTENTION**: The `id` to add the node via API (when request `/flows/v1/node`) must be the same as `name` and `id` defined in `getMetadata` in the class that extends `dojot.DataHandlerBase`. And within the html called in the `getNodeRepresentationPath` method also in the class that extends `dojot.DataHandlerBase`, the references inside the html `data-template-name=`, `data-help-name=`, `registerType(..` ,  and `RED._("...` must have this same `id`/`name`.
+#### **ATTENTION**: Avoiding problems with `id` and `name`
 
-This lib has been developed and tested using node v8.14.x
+To facilitate adding the node to the flowbroker, try to follow these intrusions:
+
+##### **When adding the node via API**
+
+The `id` when request `/flows/v1/node` must be the same as `name` and `id` defined in `getMetadata` in the class that extends `dojot.DataHandlerBase`.
```

Comment:
Suggestion:

The \`id\` sent in a request to the \`/flows/v1/node\` endpoint must be the same as the  \`name\` and the \`id\` defined in the \`getMetadata\` function inside the class that extends \`dojot.DataHandlerBase\`.